### DISCUSSION TITLE
publish package after changing user name

### DISCUSSION THREAD

**Discussion by israelstmz** [community/community#38813] [Actions]
### Select Topic Area

Bug

### Body

I recently changed my github user name. Since then, my packages publish is failing. Maven deploy keeps looking for artifacts in the URL with my old username. I updated the "distributionManagement" with the URL containing my new username but the publish action ignores it and keeps looking for my old name. I am using the standard "maven-publish.yml" action.

`name: Publish Package

on:
  release:
    types: [created]

jobs:
  build:

    runs-on: ubuntu-latest
    permissions:
      contents: read
      packages: write

    steps:
      - uses: actions/checkout@v3
      - name: Set up JDK 17
        uses: actions/setup-java@v3.6.0
        with:
          java-version: '17'
          distribution: 'temurin'
          server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
          settings-path: ${{ github.workspace }} # location for the settings.xml file

      - name: Build with Maven
        run: mvn -B package --file pom.xml

      - name: Publish to GitHub Packages Apache Maven
        run: mvn deploy -s $GITHUB_WORKSPACE/settings.xml --file pom.xml
        env:
          GITHUB_TOKEN: ${{ github.token }}`

**Comment by israelstmz** [comment]
This is the error I am getting during deploy:

`Error:  Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.7:deploy (default-deploy) on project synced-cache: Failed to deploy artifacts: Could not find artifact io.code-gems.cloud:synced-cache:pom:1.0-20221110.210336-1 in github (https://maven.pkg.github.com/israelst11/synced-in-mem-cache) -> [Help 1]`

"israelst11" is my old username...

**Comment by ossanna16** [comment]
Hi there @israelstmz and welcome to our community! Thank you for reporting this bug.
